### PR TITLE
add a codepoint for plaintextv2

### PR DESCRIPTION
**Pull request by marten-seemann**
Analogous to #221.

As we want to stop negotiating security protocols, we'll also need a codepoint for insecure connections. Example: `/ip4/1.2.3.4/tcp/4001/insecure`.

### CODE REVIEW

**Comment by Stebalien** [comment]
Do we? This is for testing only and we'll need to support security transport negotiation for the foreseeable future anyways.

**Review by Stebalien — APPROVED** [review]
Ok, so this is a real protocol and it does make sense to support it. But let's put it in the three byte range.

File: table.csv [hunk 1]
```diff
@@ -108,6 +108,7 @@ garlic64,                       multiaddr,      0x01be,         draft,     I2P b
 garlic32,                       multiaddr,      0x01bf,         draft,     I2P base32 (hashed public key or encoded public key/checksum+optional secret)
 tls,                            multiaddr,      0x01c0,         draft,
 noise,                          multiaddr,      0x01c6,         draft,
+insecure,                       multiaddr,      0x01cb,         draft,
```
Comment (mxinden) [inline table.csv:111]: When dialing a Multiaddr with the `insecure` protocol, which security protocol would be used to exchange the peer IDs? [Plaintext v2](https://github.com/libp2p/specs/tree/master/plaintext)? Why not define this as the `plaintextv2` protocol?

```suggestion
plaintextv2,                       multiaddr,      0x01cb,         draft,
```

**Review by mxinden — DISMISSED** [review]

File: table.csv (hunk 1 above)
Comment (Stebalien) [reply table.csv:111]: Yes, plaintextv2. And yeah, that might be a better name.

**Comment by marten-seemann** [comment]
I renamed the protocol to `plaintextv2`, and assigned a 3-byte code-point (PLA in ASCII, sorry for not being more creative).

**Review by Stebalien — APPROVED** [review]

**Review by mxinden — COMMENTED** [review]
:+1: thanks!

**Pull request merged**